src/context/isUserAuthorized.tsx

```diff
@@ -24,7 +24,7 @@ const IsUserAuthorizedContext: FC<Props> = ({ children }) => {
         const { adminUser, superUser } = userRoles;
         setIsUserAuthorized(!!adminUser || !!superUser);
       } catch (err: any) {
-        toast(ERROR, err.message);
+        console.error(err)
```

Comment:
Why error is ignored here?
Can't be checked for error mentioned in description and ignore only for that?

Reply:
Issue is, for "/users/self" API, user must be authenticated. The toast is shown when user is not signin.